portality/templates/doaj/contact.html

```diff
@@ -45,8 +45,16 @@
     {{ render_field_horizontal(form.message, style="width: 100%; max-width: 400px; min-height: 200px", placeholder="Enter your message here") }}
 
     <div class="form-group">
-        <div class="col-md-9 col-md-offset-3">
-            <button class="btn btn-success" id="send">Send</button>
+        <div class="submit-with-recaptcha col-md-9 col-md-offset-2">
```

Comment:
The layout here doesn't quite line up with the rest of the form.  Looks like it needs an offset of about 3, possibly with some other padding.